foqus_lib/framework/graph/node.py

```diff
@@ -45,9 +45,12 @@
     import tensorflow as tf
 
     load = tf.keras.models.load_model
-except:
-    pass  # errors will be thrown if tensorflow is called but not installed,
-    #  otherwise no error should be thrown so passing is fine
+except ImportError:
+    # if TensorFlow is not available, create a proxy function that will raise
+    # an exception whenever code tries to use `load()` at runtime
+    def load(*args, **kwargs):
+        raise RuntimeError(f"`load()` was called with args={args},"
```

Comment:
Minor suggestion, but in retrospective `ModuleNotFoundError` would be a better fit than `RuntimeError` here.